packages/sit-onyx/src/components/OnyxNavBar/modules/OnyxNavItem/OnyxNavItem.vue

```diff
@@ -11,7 +11,7 @@ const emit = defineEmits<{
   /**
    * Emitted when the nav item is clicked.
    */
-  click: [href: string];
+  navItemClick: [href: string];
```

Comment:
Preferably we use a emit name that describes the intent of the user.
So I would prefer to rename the emit to `navigate` (also for the navbutton).
```suggestion
  navigate: [href: string];
```